akka-stream/src/main/scala/akka/stream/impl/JavaStreamSource.scala

```diff
@@ -0,0 +1,41 @@
+package akka.stream.impl
+
+import akka.stream._
+import akka.stream.stage.{ GraphStage, GraphStageLogic, OutHandler }
+
+import scala.util.control.NonFatal
+
+private[stream] final class JavaStreamSource[T, S <: java.util.stream.BaseStream[T, S]](open: () ⇒ java.util.stream.BaseStream[T, S])
```

Comment:
The Scala `private[stream]` access modifier does not show in Java, annotate with @InternalApi/and add a `/** Internal API */` which will show for Java devs.